### DISCUSSION TITLE
GitHub UI could be improved

### DISCUSSION THREAD

**Discussion by julie777** [community/community#57763] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

First - I do not know why GitHub does not use their own software to manage issues and bug reports. I never know where to create an issue about how GitHub.com is working.

Issue 1

In the browser, GitHub site pages are not using screen space very well. For example, while writing this the left 1/4 of the page is blank. The right 1/3 or the page has a sidebar that takes up way more horizontal space than necessary. What is left is the useful part of the page.

![image](https://github.com/community/community/assets/5090005/2cf0b13f-8523-47d7-a4fe-a8ce7ec5eb3d)

I often want to resize the text box so that I can see more of what I am typing. I can change the vertical size, but not the horizontal size. This often causes parts of my post to be hard to review, such as inserting a section of code that does not fit in the fixed width of the editor.

Issue 2

Many times when I am ready to submit an issue, the button is off the bottom of the screen. I scroll up with my mouse to see it and click it. However, the click just scrolls the view all the way to the bottom. (I only scrolled enough to see the button.) Then I have to click the button a second time to submit.

Issue 3

Multiple times while typing this the screen was scrolled automatically, which is confusing, a visual shock, and frankly annoying. I just made it happen. I scrolled to the bottom of the page to view the buttons. My text cursor was still right here and the entire text box that I am editing was visible. The next character that I typed caused the screen to scroll so that the character I typed was on the bottom-most line of the browser window.

This is a strange behavior and incredibly bothersome. Not only the unnecessary scroll, but the result is that the bottom of the text box is not visible. Nor is the file attachment section attached to the text box and inside the boundary of the outlined area with "Write   Preview" at the top.

Issue 4

I just clicked resize handle intending to adjust the text box size and when I clicked it shrunk to a smaller size. I had not released the button. Now it will not let me resize the text box.

Coming back to this, while working on the following issues, I found that I can resize the text box smaller. It just has a max height that I cannot exceed. That begs the question "How did I get the text box to be larger than the max height before this issue happened?" An why is the text box height limited to a height less that the window height?

Issue 5

I am using a wide screen monitor. Often times I run two apps side by side, for example running an application and filling out a bug report for it in GitHub. Resizing the browser narrower for most web apps would collapse a sidebar to a hamburger menu. Instead, with GitHub this is what I get.

![image](https://github.com/community/community/assets/5090005/97afbe89-fc39-4e83-b2bb-287e76ed5715)

This makes using GitHub in a narrow window almost unusable.

**Comment by digiupdatehub** [comment]
Best Video Editing Software for Free on pc
https://www.digiupdatehub.tech/2023/06/best-video-editing-software-for-free-on.html
